lib/fugue/regula.rego

```diff
@@ -229,11 +232,65 @@ merge_reports(reports) = ret {
   }
 }
 
+# Waiver globbing.
+waiver_pattern_matches(pattern, value) {
+  pattern == "*"
+} {
+  pattern == value
+}
+
+# Should a rule-resource result be waived?
+waiver_matches_rule_result(rule_result) {
+  waiver := data.fugue.regula.config.waivers[_]
+  waiver_resource_id := object.get(waiver, "resource_id", "*")
+  waiver_rule_id := object.get(waiver, "rule_id", "*")
+  waiver_rule_name := object.get(waiver, "rule_name", "*")
+  waiver_filename := object.get(waiver, "filename", "*")
+
+  rule_result_filename := object.get(rule_result, "filename", null)
+
+  waiver_pattern_matches(waiver_resource_id, rule_result.resource_id)
```

Comment:
Could we replace this function with `glob.match(waiver_..., [], rule_result.resource_id)`?

Reply:
We discussed this on slack but noting here for posterity: going with `glob.match` gives us more flexible waivers but prevents us from using a simple dictionary to optimize this in the future, so we're going to wait and see how people use this feature first.